Source/WebGPU/WebGPU/Queue.mm

```diff
@@ -259,6 +263,11 @@ static bool validateWriteBufferInitial(size_t size)
         size:static_cast<NSUInteger>(size)];
 }
 
+bool Queue::isIdle() const
+{
+    return m_submittedCommandBufferCount == m_completedCommandBufferCount && !m_blitCommandEncoder;
```

Comment:
@litherum are you ok with this change? It fixed the race locally and EWS is green.